cgi-bin/DW/Worker/XPostWorker.pm

```diff
@@ -62,6 +62,22 @@ sub work {
     my $acct = DW::External::Account->get_external_account( $u, $acctid )
         or return $job->failed("Unable to load account $acctid for uid $uid");
 
+    my $sclient = LJ::theschwartz();
+    my $notify_fail = sub {
+        $sclient->insert_jobs(
+            LJ::Event::XPostFailure->new(
+                $u, $acctid, $ditemid, ( $_[0] || 'Unknown error message.' )
+            )->fire_job
+        );
+    };
+
+    # LJRossia is temporarily broken, so skip - but we do want to notify
+    if ( $acct->externalsite && $acct->externalsite->{sitename} eq 'LJRossia' ) {
```

Comment:
Wow, is this really the best way of doing this?! Hardcoding like this feels pretty sad, but I guess the original design of the xpost system didn't take into account that we might want to administratively disable things? I don't recall thinking of this use case :)